packages/react/src/components/ListBox/ListBoxMenuItem.tsx

```diff
@@ -63,6 +68,7 @@ const ListBoxMenuItem = React.forwardRef<HTMLLIElement, ListBoxMenuItemProps>(
       children,
       isActive = false,
       isHighlighted = false,
+      isSelectAll = false,
       title,
```

Comment:
I noticed we have `title` and `children` on the ListBoxMenuItem.propTypes but no in the interface ListBoxMenuItemProps. We could also add it there

Reply:
Could be something to defer for #6177